cannot npm install conflicting react dependencies

#### Description
```console
$ npm install
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: nextjs-notion-starter-kit@2.0.0
npm ERR! Found: react@18.2.0
npm ERR! node_modules/react
npm ERR!   react@"^18.2.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^16.8.0  || ^17.0.0" from react-use@17.3.2
npm ERR! node_modules/react-use
npm ERR!   react-use@"^17.3.2" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /Users/user/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/user/.npm/_logs/2023-09-04T18_27_17_720Z-debug-0.log
```
<!--
Please include as detailed of a description as possible, including screenshots if applicable.
-->

#### Notion Test Page ID

<!--
Please include the ID of at least one publicly accessible Notion page related to your issue.

This is extremely helpful for us to debug and fix issues.

Thanks!
-->

--- a/package.json
+++ b/package.json
@@ -49,7 +49,7 @@
     "react-dom": "^18.2.0",
     "react-notion-x": "^6.15.6",
     "react-tweet-embed": "^2.0.0",
-    "react-use": "^17.3.2",
+    "react-use": "^17.4.0",
     "rss": "^1.2.2"
   },
   "devDependencies": {

